docs/modules/overview/getting-started.md

```diff
@@ -30,4 +32,4 @@ The template will attempt to auto-fill details in your manifest as well as defin
 3. Ensure the Blish HUD and all of Blish HUD's dependent references have **Copy Local** set to **False** in the properties of the reference.
```

Comment:
Looks like maybe this number didn't get updated?

Reply:
That's a different list actually, if you're referring to the 3.

Reply:
Oh!